### PR TITLE
Run pytest after package installation

### PR DESCRIPTION
**Pull request by ndrezn**
For some reason on `release.yml`, tests were being run before packages were installed.

### CODE REVIEW

**Review by gvwilson — APPROVED** [review]

File: package.json
Comment (BSd3v) [inline package.json:1]: Eh, I dont know about this.

This assumes that the dev wants to run tests before creating a distro. For example, what if I wanted to install it onto an app using `npm run dist`, I'd have to sit through the tests again.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -24,15 +24,10 @@ jobs:
...
-          uv pip install ".[dev]"
-          pytest --headless
       - name: Generate distribution
         run: |
           source .venv/bin/activate
-          uv pip install -r requires-dev.txt -r requires-install.txt
+          uv pip install ".[dev]"
           npm i
           npm run dist
           npm pack && mv *.tgz dist/
```
Comment (BSd3v) [inline .github/workflows/release.yml:33]:
```
npm i
npm run dist
pytest --headless
npm pack && mv *.tgz dist/
```

I'd just add the test there.

**Review by BSd3v — CHANGES_REQUESTED** [review]

File: package.json
Comment (ndrezn) [reply package.json:1]: I'll make it an optional step. I don't think we ever want to allow generating a dist without passing tests but we can enforce it in CI anyway.

**Review by BSd3v — APPROVED** [review]
💃

**Pull request merged**